Simplenote/Classes/NSTextStorage+Highlight.m

```diff
@@ -30,49 +22,15 @@ - (void)applyColorAttribute:(id)color forRanges:(NSArray *)wordRanges {
         
         // Out of Range Failsafe
         NSRange range = rangeValue.rangeValue;
-        if (range.location + range.length >= maxLength) {
+        if (range.location + range.length > maxLength) {
```

Comment:
You could write `NSMaxRange(range) > maxLength`

Reply:
Thank you!! I wasn't aware of this one!!